content/docs/1_guide/18_plugins/1_plugin-basics/guide.txt

```diff
@@ -85,6 +93,18 @@ Check out (link: docs/guide/plugins/plugin-types text: the list of all extension
 
 If you need to register extensions based on a config option, site content or even the availability of other plugins, you can do this inside a (link: docs/reference/plugins/hooks/system-loadplugins-after text: `system.loadPlugins:after` hook).
 
+### Panel-only plugins
```

Comment:
Why are we adding this? The 3.7.1 changes kinda make this not necessary anymore.

Reply:
Instead we probably need to go into e.g. sections docs and remove this superfluous code.

Reply:
Not necessary, but see above where it says we still recommend it to register the plugin with a name instead of anonymously? So this clarifies how to do it properly.

Reply:
Yes, especially because of the new plugin table in System view, it still makes sense for published plugins. Even more if we add update checks or similar later.